bin/main.ml

```diff
@@ -177,7 +191,7 @@ let draw ui_terminal matrix (game_state : game_state) =
     (* draw players box *)
     (let ctx = with_grid_cell ctx layout_spec 0 1 0 1 in
      let _ = with_frame ctx " players " LTerm_draw.Heavy in
-     ());
+     draw_players ctx game_state.players);
```

Comment:
Implementing the function change above causes this to be too messy. I suggest adding another `let` statement at around line 185.

Before the `let current_player = ` add a ` let players = game_state.players in ` 
then change line the `current_player` declaration to `let current_player = List.hd players in `

Then change this line to ` draw_players ctx (List.length players) players `